model_helper.py

```diff
@@ -129,6 +129,14 @@ def compute_loss_sigmoid(logits, targets, final_sequence_length, target_sequence
 
     return loss
 
+def compute_log_probs_loss(outputs):
+    nfeatures = outputs.shape[-1] // 2
+    log_prob_ones = outputs[..., :nfeatures]
+    log_prob_zeros = outputs[..., nfeatures:]
+    norm_const = tf.stop_gradient(-(log_prob_ones + log_prob_zeros) / 2)
+    loss = tf.abs((tf.exp(log_prob_ones + norm_const) + tf.exp(log_prob_zeros + norm_const)) / tf.exp(norm_const) - 1)
+    loss += tf.nn.relu(log_prob_ones) + tf.nn.relu(log_prob_zeros)
```

Comment:
This one tries to enforce `0 <= P <= 1` constraint? I think we might need to weight this part of loss with some regularization parameter.

Reply:
Yes. I can add weight to the arguments, and set 1 by default, and later experiment with different values.